testing/scripts/test_api.sh

```
@@ -757,6 +757,131 @@ else
     skip "Site hierarchy tests — could not create parent region (HTTP $HTTP_CODE)"
 fi
 
+# ====================================================================
+log "=== VRFs ==="
+# ====================================================================
+
+call_api GET vrfs
+assert_http 200 "GET vrfs → 200"
+VRFS_ARRAY=$(python3 -c "import sys,json; print(type(json.load(sys.stdin).get('vrfs',[])).__name__)" <<< "$BODY" 2>/dev/null || echo "")
+[[ "$VRFS_ARRAY" == "list" ]] && pass "vrfs array is a list" || fail "vrfs response missing 'vrfs' array"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)